dev-python/moviepy/moviepy-0.2.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Video editing with Python"
+HOMEPAGE="http://zulko.github.io/moviepy/"
+SRC_URI="https://github.com/Zulko/moviepy/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	"
```

Comment:
> do I need to define python_test() explicitly?

Yes, see how distutils-r1 testing machinery works: https://github.com/gentoo/gentoo/blob/master/eclass/distutils-r1.eclass#L783
```
python_test() {
    py.test --verbose || die
}
```
is the right way, because testsuite potentially can grow scattered across many files.